pkg/api/schema/schema.proto

```diff
@@ -451,6 +454,16 @@ message TxList {
 
 message ExportTxRequest {
 	uint64 tx = 1;
+	bool allowPreCommitted = 2;
+	FollowerState followerState = 3;
+}
+
+message FollowerState {
+	string UUID = 1;
```

Comment:
Currently we don't share UUID across primary and replicas, but I assume this is rather some replica's UUID rather than cluster UUID?

Reply:
it's the follower UUID

Reply:
Something to keep in mind then - once we introduce a notion of a cluster - such cluster should be seen as a single UUID which means we will have to differentiate between immudb process UUID and immudb cluster UUID.